### PR TITLE
fixes basic mobs playing body drop sfx

### LINKED ISSUE #88426: basic mobs (i think?) make the body falling sound on death
### Client Version:

_No response_

### Issue Summary:

basic mobs (i think?) make the body falling sound on death
this is kinda just dumb for all cases anyway but most prevalently you can get this by stepping on a cockroach

### Round ID:

_No response_

### Test Merge Information:

_No response_

### Reproduction Steps:

1. step on cockroach and crush it
2. hear body fall sfx

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
closes https://github.com/tgstation/tgstation/issues/88426
## Changelog
:cl: grungussuss
fix: basic mobs don't play body drop sfx on death anymore
/:cl:

### CODE REVIEW

**Comment by mc-oofert** [comment]
cant you just exclude it via type check or something

File: code/modules/mob/living/basic/basic.dm [hunk 1]
```diff
@@ -100,6 +100,7 @@
 
 /mob/living/basic/Initialize(mapload)
 	. = ..()
+	ADD_TRAIT(src, TRAIT_NO_BODY_DROP_SFX)
```
Comment (Ghommie) [inline code/modules/mob/living/basic/basic.dm:103]:
```suggestion
	ADD_TRAIT(src, TRAIT_NO_BODY_DROP_SFX, INNATE_TRAIT)
```

**Comment by mc-oofert** [comment]
I think a trait for this is dumb

File: code/__DEFINES/traits/declarations.dm [hunk 2]
```diff
@@ -515,6 +515,9 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_RIGHT_EYE_SCAR "right_eye_scar"
 #define TRAIT_LEFT_EYE_SCAR "left_eye_scar"
 
+/// Mob doesn't play body drop sfx when dying/in crit
+#define TRAIT_NO_BODY_DROP_SFX "NO_BODY_DROP_SFX"
```
Comment (LemonInTheDark) [inline code/__DEFINES/traits/declarations.dm:519]: this doesn't make any sense to me it's applied on init. also the motivation doesn't seem tied to it at all

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]
pull and update tests are failing

**Pull request closed without merging**